envs.json

```diff
@@ -94,6 +94,28 @@
       "BMp6gEnveANdvSvspESJUrNczuHz1GF5UQKjVLCkAZih"
     ]
   },
+  "mainnet": {
+    "evm_loader": "NeonVMyRX5GbCrsAHnUwx1nYYoJAtskU1bWUo6JGNyG",
```

Comment:
we can add one or two tests for erc20 and many tests from rpc folder - many of them are free
also one or two tests from transfer folder

Reply:
Ok, will do